docs/designers-developers/developers/tutorials/create-block/wp-plugin.md

```diff
@@ -78,13 +78,13 @@ To use the scripts package, the `scripts` property of package.json defines the p
   },
 ```
 
-These are run using: `npm run build` or `npm run start` which will call the appropriate binary and command we need.
+Sp these scripts are run by using: `npm run build` or `npm run start`
```

Comment:
`npm start` - it’s one of special cases, `npm test` is another one

Reply:
True, but that is a shortcut and works both ways. So mapping each command to `npm run` is clearer than trying to explain special ones. Something for them to pick up later.